src/Api/Services/Projects/ProjectsApiProcessor.php

```diff
@@ -29,6 +40,95 @@ public function saveProject(Program $project): void
     $this->project_manager->save($project);
   }
 
+  public function updateProject(Program $project, UpdateProjectRequest $request): bool|int
```

Comment:
```suggestion
  public function updateProject(Program $project, UpdateProjectRequest $request): int
```

Wär in dem Fall nicht ein neuer Status Code, zbsp 0, sinnvoller als ein return true? Ist vermutlich Geschmacksache, aber ich vermeide am liebsten mixed return types wenn immer möglich.